guides/dev/issues_readme/README.md

```diff
@@ -17,7 +17,9 @@ Most of the issues are reproducible on the demo. If you have a different case th
 
 ## Avoid Duplicates
 
-Before you file a new ticket, make sure your issue has not been reported yet. Use the [search page](https://dev.ckeditor.com/search) to check it.
+Before you file a new ticket, make sure your issue has not been reported yet. Use the [search page](https://github.com/ckeditor/ckeditor-dev/issues) to check it.
```

Comment:
How about "before you file a new **issue**". GH uses word issue rather than ticket - so it might be better to stick with it.